### DISCUSSION TITLE
为什么转图片会比直接对PDF进行OCR，识别效率更高？

### DISCUSSION THREAD

**Discussion by TeeDubDub** [hiroi-sora/Umi-OCR#435] [Q&A] [answered]
试了着对一个扫描公文进行了OCR，发现转成图片的识别正确率更高一些，是因为对PDF的识别，采集密度低吗？
另外咱们有没有互助讨论群，想加一下学习学习

**Comment by hiroi-sora** [comment, marked as answer]
### 1. 转成图片的识别正确率更高一些

OCR受到多方面的影响。对于同一张图片，即使小幅修改分辨率、边缘裁切、文件编码等，都可能造成OCR结果的区别。

默认情况下，Umi-OCR从PDF中直接提取出图像源数据。而你的做法是将PDF整页渲染为图片文件，即对源数据做了调整。这种调整说不准会带来正面或负面的影响。比如你的例子里，识别准确率提高了；但可能相同的做法对于另一个文件则有负面效果。如果想实现最优效果，可能并没有通用的做法，只能对具体文件尝试不同做法。

其实Umi-OCR也支持先将PDF转为图片、再进行OCR的方案：`标签页设置→内容提取模式→整页强制OCR`。（不过该方案在现版本下有缺陷，将在下版本完善。）

### 2. 关于交流群

本项目目前主要是我一个人业余时间开发更新。精力有限，暂时懒得维护那么多社交圈。有问题请在Github找我吧。
